app.rb

```diff
@@ -0,0 +1,48 @@
+require 'book'
+require 'person'
+require 'rental'
+require 'teacher'
+require 'student'
+
+class App
+  def list_all_books(books)
+    books.each do |i|
+      puts(" Title: \"#{i.title}\", Author : #{i.author}")
+    end
+  end
+
+  def list_all_people(people)
+    people.each do |i|
+      stud = '[Student]'
+      if defined?(i.specialization)
+        stud = '[Teacher]'
+      end
+      puts("#{stud} Name: #{i.name}, ID : #{i.id} ,Age : #{i.age}")
+    end
+  end
+
+  def create_people(age, name, parent_permission, cat, specialization)
+    if cat == 1
+      classr = Classroom.new('1ere')
```

Comment:
- [ ] Am not able to create a person yet, only when selecting student and not teacher, does this error below happens, please check the image and fix the issue
<img width="630" alt="1" src="https://github.com/francksefu/school_library_oop/assets/77843050/69baab07-84c0-4134-9c9a-7ac62d0b19cd">